README.md

```diff
@@ -367,6 +368,15 @@ final class SecondaryVC: UIViewController, StoryboardSceneBased {
 ```
 </details>
 
+<summary>📑 Example of a ViewController being the root view of its xib</summary>
+
+In this example, `CustomVC` is designed as the root view of a xib named `CustomVC.xib`:
```

Comment:
That's not true though, a `UIViewController` designed in a XIB is expected to be the XIB's **File's Owner**, not the **root view** of the XIB (a `UIViewController` isn't even an `UIView`, it can't be the root _view_…)

It's very important to document this right because this is a key difference that many users are not aware of, and it's also the key difference between `NibLoadable` (no File's Owner, view is the root view, as it's the case for `UITableViewCells`) and `NibOwnerLoadable` (the class (here the `UIViewController` is the File's Owner and is already `init` when it decides to load its `UINib`  via `nib.instantiate(owner: thatAlteradyInstantiatedOwner, options: …)`  internally — in the case of a `UIViewController`, that's done when the UIViewController needs to load its root `view` from said XIB)

All that being said, in retrospect **it would make more sense to make XIB-based UIViewControllers work using `NibOwnerLodable` and not `NibLoadable`**, as by definition and semantics, `NibLoadable` expects no File's Owner and the instance to load to be the first and only root item of the XIB, while the way to design XIB-based UIViewController is following the semantics of `NibOwnerLoadable`, with the class to instantiate being set as the File's Owner of the XIB.